app/models/event.rb

```diff
@@ -82,7 +94,7 @@ def calculate_round_robin
         matches << Match.new(team_home: pairing[0], team_away: pairing[1], gameday: gameday + 1) unless pairing[0].nil? or pairing[1].nil?
       end
     end
-    save
+    self.save
```

Comment:
Is there any reason for this? Do our coding standard include using self even when it isn't necessary? Because I don't think we should use it if it isn't needed. And if we do we should at least do it everywhere.